sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Utils.java

```diff
@@ -777,4 +777,12 @@ public static long getMaxIntegratedCacheStalenessInMillis(DedicatedGatewayReques
         }
         return maxIntegratedCacheStaleness.toMillis();
     }
+
+    public static String toJsonString(Object input) {
+        try {
+            return getSimpleObjectMapper().writeValueAsString(input);
+        } catch (JsonProcessingException jsonError) {
+            return "EXCEPTION: " + jsonError.toString();
```

Comment:
Why are we returning Exception string here. 

Shouldn't we log the exception and just return null string. So that it can be handled gracefully. 
How does returning the Exception string message helps here ?

Reply:
Good catch, updated.